cmd/cloud-controller-manager/main.go

```diff
@@ -20,27 +20,70 @@ limitations under the License.
 package main
 
 import (
-	"k8s.io/cloud-provider-gcp/cmd/cloud-controller-manager/app"
 	"math/rand"
 	"os"
 	"time"
 
+	"github.com/spf13/pflag"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/cloud-provider"
+	"k8s.io/cloud-provider/app"
+	"k8s.io/cloud-provider/app/config"
+	"k8s.io/cloud-provider/options"
+	cliflag "k8s.io/component-base/cli/flag"
 	"k8s.io/component-base/logs"
-	_ "k8s.io/component-base/metrics/prometheus/version" // for version metric registration
-	// NOTE: Importing all in-tree cloud-providers is not required when
-	// implementing an out-of-tree cloud-provider.
-	_ "k8s.io/cloud-provider-gcp/providers/gce"
 	_ "k8s.io/component-base/metrics/prometheus/clientgo" // load all the prometheus client-go plugins
+	_ "k8s.io/component-base/metrics/prometheus/version"  // for version metric registration
+	"k8s.io/klog/v2"
+
+	_ "k8s.io/cloud-provider-gcp/providers/gce"
+	// For existing cloud providers, the option to import legacy providers is still available.
```

Comment:
These two comment lines should be removed.

Reply:
Updated